bin/sympy_time_cache.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import print_function
+from sympy.core.compatibility import string_types
```

Comment:
Revert the change to this file. It should not import SymPy before the import at the bottom. I believe the instance where `str` is used can only be a str.